libraries/AP_Math/vector3.cpp

```diff
@@ -251,9 +251,13 @@ void Vector3<T>::rotate(enum Rotation rotation)
         z = -sin_pitch * tmpx + cos_pitch * tmpz;
         return;
     }
-    case ROTATION_CUSTOM: // no-op; caller should perform custom rotations via matrix multiplication
```

Comment:
We've lost this hint to the poor sod who's just had his code fall over... could we merge this onto line 256?